### PR TITLE
Fixing delete in drafts+ mass edit mode

### PR DESCRIPTION
**Pull request by blackjackkent**

### CODE REVIEW

File: Extensions/drafts_plus.js [hunk 1]
```diff
@@ -259,8 +259,12 @@ XKit.extensions.drafts_plus = new Object({
 			GM_xmlhttpRequest({
 				method: "POST",
 				url: "http://www.tumblr.com/svc/post/delete",
-				data: JSON.stringify(m_object),
-				json: true,
+				headers: {
+					"Content-Type": "application/x-www-form-urlencoded",
+					"X-tumblr-form-key": XKit.interface.form_key(),
+				},
+				data: "post_id=" + m_object.post_id + "&channel_id=" + m_object.channel_id,
```
Comment (nightpool) [inline Extensions/drafts_plus.js]: this should be `$.param`, right?

Comment (blackjackkent) [inline Extensions/drafts_plus.js]: That works too. I was following the pattern already existing in the other calls; I'll update those as well.

**Comment by hobinjk** [comment]
Tested locally, LGTM. Can this fix also be applied to Mass Deleter? ([like here maybe?](https://github.com/new-xkit/XKit/blob/master/Extensions/mass_deleter.js#L172))

**Comment by nightpool** [comment]
LGTM @homu r+

**Comment by nightpool** [comment]
@hobinjk can you file or bump an issue for that one?

**Pull request merged**